lib/plugins/inspec-compliance/lib/inspec-compliance/cli.rb

```diff
@@ -13,10 +13,9 @@ class CLI < Inspec.plugin(2, :cli_command)
       long_desc <<-LONGDESC
         `login` allows you to use InSpec with #{AUTOMATE_PRODUCT_NAME} Server
 
-        You need to a token for communication. More information about token retrieval
+        You need to have a token for communication. More information about token retrieval
         is available at:
-          https://docs.chef.io/api_automate.html#authentication-methods
-          https://docs.chef.io/api_compliance.html#obtaining-an-api-token
+          https://docs.chef.io/automate/api/#tag/tokens
```

Comment:
Is there any user-level documentation for this? The URL given above is for API documentation, which is not what we want to send users to. If there is no suitable URL, we should remove the URL.

Reply:
That makes sense. 
How about [https://docs.chef.io/automate/api_tokens](https://docs.chef.io/automate/api_tokens) ? This is not the API doc but it might contain similar kind of info too. Does this make sense?